android-components/components/feature/app-links/src/main/java/mozilla/components/feature/app/links/AppLinksFeature.kt

```diff
@@ -95,7 +100,30 @@ class AppLinksFeature(
             return
         }
 
+        val currentTimeStamp = SystemClock.elapsedRealtime()
```

Comment:
nit: could we move this declaration to be above the `cacheTimeStamp` declaration since it isn't used until about then? Would keep it more closely grouped with the rest of the timestamp logic

Reply:
Good idea.  I'll move it, thanks